packages/react/src/Card/Card.features.stories.tsx

```diff
@@ -120,20 +121,21 @@ export const IconColors: StoryFn<typeof Card> = () => {
 export const WithIconSVG = () => (
   <Card href="https://github.com">
     <Card.Icon
-      icon={
+      icon={props => (
```

Comment:
Is `props` doing something specific here or can we keep it the same as before?

Reply:
`props` includes the classNames, without it the Icon styles won't be applied to the svg

Reply:
I'm guessing this was needed because we only accepted function references? What would happen if someone forgot to spread the props?

For now, I've rolled this back to allow passing the SVG directly as a prop. We can figure out if the props argument is still needed when you're back.